src/wp-includes/theme-templates.php

```diff
@@ -203,6 +203,7 @@ function wp_enqueue_block_template_skip_link() {
 		// Create the skip link.
 		skipLink = document.createElement( 'a' );
 		skipLink.classList.add( 'skip-link', 'screen-reader-text' );
+		skipLink.id = 'skip-link';
```

Comment:
```suggestion
		skipLink.id = 'wp-skip-link';
```

`wp-skip-link` might help avoid duplicate IDs

Reply:
Fixed